2200095/problem6.py

```diff
@@ -0,0 +1,11 @@
+# 5431. 민석이의 과제 체크하기
+tc = int(input())
+for t in range(tc):
+    N, K = map(int, input().split())
+    res = [x for x in range(1, N + 1)]
```

Comment:
`remove` 함수 활용이 인상적이네요! 👍 
```suggestion
    res = list(range(1, N + 1))
```
바로 `list`로 바꿔주셔도 똑같이 동작합니다 😄